examples/topological-sort/webpack.config.js

```diff
@@ -0,0 +1,53 @@
+var plugin = require('../../index.js');
+var webpack = require('webpack');
+var _ = require('lodash');
+var toposort = require('toposort');
+
+function generate(seed, files) {
+  var nodeMap = {};
+
+  files.forEach(function (file) {
+    nodeMap[file.chunk.id] = file;
+  });
+
+  var edges = [];
+
+  files.forEach(function (file) {
+    if (file.chunk.parents) {
```

Comment:
same here

Reply:
so i'm not actually all that familiar with webpack, what kind of dependency tree would give me files without chunks? I supposed the example should handle that case? or is it sufficient to handle the actual case that the example files dep tree creates (all files are chunks?)

Reply:
Oops sorry, missed that comment.

We include emitted files (for example from `copy-webpack-plugin`), basically it's every non-js files. I think those better be at the end (but really I don't care, up to you!)